tor_core/initialize.py

```diff
@@ -230,7 +230,7 @@ def build_bot(name, log_name='transcribersofreddit.log', require_redis=True):
         # I'm sorry
```

Comment:
I figured it was something that expected a callable value, e.g., a lambda or a method reference. I was a little thrown by the generator part, but it makes more sense now. :+1: